packages/capabilities/src/types.ts

```diff
@@ -514,6 +515,21 @@ export type AggregateAccept = InferInvokedCapability<
   typeof DealerCaps.aggregateAccept
 >
 export type DealInfo = InferInvokedCapability<typeof DealTrackerCaps.dealInfo>
+
+// Plan
+
+export type PlanGet = InferInvokedCapability<typeof PlanCaps.get>
+export interface PlanGetSuccess {
+  updatedAt: string
```

Comment:
What kind of string? Reading the code it looks like this will usually be an ISO8601 datetime string.
* I think that's a great choice
* notably, it is different than the number used in https://github.com/web3-storage/w3up/issues/959 . But again, I prefer a string to a number here
* string/number aside, here in the `PlanGetSuccess` typescript type, IMO it's worth adding a comment explaining constraints like ISO8601 string
  * may also be testing this in relevant tests that receive a PlanGetSuccess, i.e. can the PlanGetSuccess#updatedAt value be parsed as ISO8601 to a realistic datetime

Reply:
ah good call, agreed! I've left a comment on #959 to this effect, and have created a new `string` alias called `ISO8601Date` so that this is clearer at least. also added a test to the handler that verifies we keep dates in 8601 format throughout the stack.